rebar.config

```diff
@@ -10,7 +10,7 @@
 
 {deps, [
         {blockchain, {git, "https://github.com/helium/blockchain-core.git",
-                      {branch, "master"}}},
+                      {branch, "rg/nat-fixes"}}},
```

Comment:
Switch this back to master when https://github.com/helium/blockchain-core/pull/728 lands